docs/components/detail/abbreviations.html

```diff
@@ -3256,6 +3267,35 @@ <h4 class="Tree-collectionLabel" data-role="toggle">
         
     
         
+        
+        <li class="Tree-item Tree-entity" data-role="item">
+            <a class="Tree-entityLink" href="side-nav--alternative.html" data-pjax>
+                <span>Alternative</span>
+                
+
+<div class="Status Status--unlabelled">
```

Comment:
@adrianconte could you look at the formatting of this section and update accordingly in each of the relevant files.